.github/workflows/static-checks.yml

```diff
@@ -124,7 +133,15 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: '8'
-          cache: 'maven'
+
+      # restore maven dependencies only. 'depsonly' keys are safe to restore
```

Comment:
yes - this will restore it ; but never save it - is that intentional?

Reply:
we are saving them as part of the `static checks` job above.